unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTF/AssetPromise_GLTF.cs

```diff
@@ -14,6 +15,10 @@ public class AssetPromise_GLTF : AssetPromise<Asset_GLTF>
         GLTFComponent gltfComponent = null;
         object id = null;
 
+        private Action OnSuccess;
+        private Action OnFail;
```

Comment:
```suggestion
```
Why are we copying the parent class `OnSuccess` value? I don't see this copy having any purpose.

Reply:
If we are copying it to prevent it to being modified while the asset is being loaded, we can just make the copy inside the method as a local variable.

Reply:
Those are just to be able to clean them. Those are not a copy of the parent class. Those are the actions received as arguments on `OnLoad` method. I don't want the `GLTFComponent` triggering `OnLoad` callbacks if the promise was forgotten, therefor I need to clean them